app/services/proxy_status/dhcp.rb

```diff
@@ -0,0 +1,20 @@
+module ProxyStatus
+  class DHCP < Base
+
+    def subnets
+      return @subnets if @subnets
+
+      fetch_proxy_data do
+        @subnets = api.subnets.map do |subnet|
+          records = api.subnet(subnet['network'])
```

Comment:
this will be potentially slow with many subnets, the alternative is to create another controller serving subnets (as similar to the dhcp proxy plugin) what do you think @xprazak2 ?